Sources/Lumina/UI/LuminaButton.swift

```diff
@@ -84,10 +84,7 @@ final class LuminaButton: UIButton {
         self.frame = CGRect(origin: CGPoint(x: UIScreen.main.bounds.maxX - 50, y: 10), size: CGSize(width: self.squareSystemButtonWidth, height: self.squareSystemButtonHeight))
       case .shutter:
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dokun1/Lumina/blob/8cefb8053a7a558007ec4a93a1a2b01854a11a38/Sources/Lumina/UI/LuminaButton.swift#L85"></span>Case statements should vertically align with their enclosing switch statement.<br />
<code>switch_case_alignment</code> <code>LuminaButton.swift:85</code></td>
    </tr>
  </tbody>
</table>